clusters/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml

```diff
@@ -32,17 +42,17 @@ spec:
   - name: openstack-k8s-operators-cdc-vexx-19
   labels:
     tp.openshift.io/owner: openstack-k8s-operators
-  maxConcurrent: 9
-  maxSize: 9
+  maxConcurrent: 19
+  maxSize: 19
   platform:
     openstack:
       cloud: openstack
       credentialsSecretRef:
         name: openstack-operators-vexxhost-hive-creds
   pullSecretRef:
     name: pull-secret
-  runningCount: 9
-  size: 9
+  runningCount: 15
```

Comment:
ok, 15 is fine

Reply:
Yeah I set to 15 as that's what 4.12 had before we started this migration :)